### ISSUE TITLE
Linux/x86 Support

### ISSUE THREAD

**Issue by rkeene** [codercom/code-server#14]
I'd like a Linux/x86 binary so I can run this on my Linux/x86 box

**Comment by sr229** [comment]
Already exists.

**Comment by nhooyr** [comment]
@scsmithr pointed out that the user actually means 32 bit support for which we do not have binaries up.

**Comment by rkeene** [comment]
Correct, x86 is a 32-bit platform.  x86_64/amd64 are the 64-bit extensions to that 32-bit platform from AMD.  IA64 is the Intel (now failed) attempt at a new 64-bit platform (Itanium/Itanium2).

**Comment by WuLC** [comment]
Awesome tool, hope that binaries for 32-bit Linux coming soon

**Comment by JordanStuckings** [comment]
Awesome tool. I would also love 32-bit binaries. Thanks for the great work so far.

**Comment by germanllop** [comment]
Hello. Can I compile it on a 32bit Linux Machine?

**Comment by kylecarbs** [comment]
@germanllop you should be able to compile on 32bit. You'll have to compile our binary from `nbin` for a 32bit target first.
